package-lock.json

```diff
@@ -1,8 +1,14126 @@
 {
   "name": "@financial-times/n-express",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
**issue**: this lockfile was generated with npm 7, which we don't fully support yet. could you revert this change please?

Reply:
Whoops, will remove.